.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,109 @@
+name: Bug report
+description: Use this template for reporting bugs/issues.
+labels: [ "kind/bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: act-version
+    attributes:
+      label: Act version
+      description: |
+        What version of `act` are you using? Version can be obtained via `act --version`
+        If you've built it from source, please provide commit hash
+      placeholder: |
+        act --version
+    validations:
+      required: true
+  - type: textarea
+    id: docker-version
+    attributes:
+      label: Docker version
+      description: |
+        What version of `docker` are you using? Run `docker version` to get it
+      placeholder: |
+        output of command: docker version
+      render: sh
+    validations:
+      required: false
+  - type: textarea
+    id: actrc
+    attributes:
+      label: .actrc content
+      description: |
+        Output of `~/.actrc` and `./.actrc`?
+      placeholder: |
+        Skip if you don't have/use .actrc
+      render: sh
+    validations:
+      required: false
+  - type: textarea
+    id: act-command
+    attributes:
+      label: Command used with act
+      description: |
+        Please paste your whole command
+      placeholder: |
+        act -P ubuntu-latest=node:12 -v -d ...
+      render: sh
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe issue
+      description: |
+        Also tell us what did you expect to happen?
+      placeholder: |
+        Describe issue
+    validations:
+      required: true
+  - type: input
+    id: repo
+    attributes:
+      label: Link to GitHub repository
+      description: |
+        Provide link to GitHub repository, you can skip it if the repository is private or you don't have it on GitHub, otherwise please provide it as it might help us troubleshoot problem
+      placeholder: |
+        https://github.com/nektos/act
+    validations:
+      required: false
+  - type: textarea
+    id: workflow
+    attributes:
+      label: Workflow content
+      description: |
+        Please paste your **whole** workflow here
+      placeholder: |
+        name: My workflow
+        on: ['push', 'schedule']
+        jobs:
+          test:
+            runs-on:: ubuntu-latest
```

```suggestion
            runs-on: ubuntu-latest
```
Just a small typo in a placeholder